deforestation_api/routers/deforestation.py

```diff
@@ -0,0 +1,67 @@
+from shapely import Point
+import geopandas as gpd
+import pandas as pd
+import json
+
+from fastapi import APIRouter
+
+from deforestation_api.dependencies.deforestationdata import (
+    BasinDataDep,
+    LossyearDataDep,
+)
+from deforestation_api.dependencies.queryparams import CoordinatesDep, DateRangeDep
+
+router = APIRouter(prefix="/deforestation", tags=["deforestation"])
```

Comment:
The `prefix="/deforestation"` was removed from the flood-api, I think it can be removed here as well?

Reply:
Yes, we can remove it from here as well. It will be added when deployed to the cluster, but stripped before the request is forwarded to the api.